node/alphanet/src/lib.rs

```diff
@@ -46,10 +46,6 @@ pub fn start_from_client(
     let node_task = futures::lazy(move || {
         spawn_rpc_server_task(client.clone(), &rpc_cfg);
 
-        // Receipts from previous blocks.
-        let (receipts_tx, receipts_rx) = channel(1024);
-        spawn_receipt_task(client.clone(), receipts_rx);
```

Comment:
you should remove this function as well below

Reply:
Done.